README.md

```diff
@@ -237,61 +239,46 @@ the output to a log file or other log management system.  See the
 [signalfx-agent command](./docs/signalfx-agent.1.man) doc for more information on
 supported command flags.
 
-#### Windows Standalone .zip
-If you don't want to use the installer script, we offer a
-.zip that can be deployed to the target host.  This bundle is available for
-download on the [Github Releases
-Page](https://github.com/signalfx/signalfx-agent/releases) for each new
-release.
-
-Before proceeding make sure the following requirements are met.
-- [.Net Framework 3.5](https://docs.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows-10) (Windows 8+)
-- [Visual C++ Compiler for Python 2.7](https://www.microsoft.com/EN-US/DOWNLOAD/DETAILS.ASPX?ID=44266)
-
-To use the bundle:
+#### Windows Chocolatey Package
+_Only available for Smart Agent v5.3.0 or later._
 
-1) Unzip it to a directory of your choice on the target system.
+If [Chocolatey](https://chocolatey.org/) is installed on your Windows system, you
+can install the agent by running the following Powershell command as an
+Administrator:
 
-2) Ensure a valid configuration file is available somewhere on the target
-system.  The main thing that the installer script provides -- but that you will
-have to provide manually with the bundle -- is a run directory for the agent to
-use.  Since you aren't installing from a package, there are two config
-options that you will especially want to consider:
-
- - `internalStatusHost` - This is the host name that
-	 the agent will listen on so that the `signalfx-agent status` command can
-	 read diagnostic information from a running agent.  This is also the host name the
-	 agent will listen on to serve internal metrics about the agent.  These metrics can
-	 can be scraped by the `internal-metrics` monitor.  This will default to `localhost`
-	 if left blank.
+```sh
+choco install signalfx-agent --params "'/access_token:YOUR_SIGNALFX_API_TOKEN'"
```

Comment:
use `<ACCESS_TOKEN>`. It has to be an org token (access token). We don't use the term API_TOKEN anywhere in the developer documentation, and in the user documentation we refer a **session token** User Access API Token. Session tokens won't work.